www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -80,47 +78,79 @@
         }
         $statement->bindValue(':value', $fileName, \PDO::PARAM_STR);
         $statement->bindValue(':id', $row['bk_mod_id'], \PDO::PARAM_INT);
-        $statement->execute();
+        if (false === $statement->execute()) {
+            throw new \PDOException(
+                $versionOfTheUpgrade . "Unable to move engine's broker modules configuration from xml to json format"
+            );
+        };
     }
-} catch (\PDOException $e) {
-    $centreonLog->insertLog(
-        2,
-        "UPGRADE : 20.04.0-beta.1 Unable to move engine's broker modules configuration from xml to json format"
-    );
-    throw new \PDOException($e);
-}
 
+    /**
+     * Change broker sql output form
+     */
+    // set common error message on failure
+    $errorMessage = $versionOfTheUpgrade .
+        "Unable to move engine's broker modules configuration from xml to json format";
 
-// Change broker sql output form
-
-try {
     // reorganise existing input form
-    $pearDB->query(
-        "UPDATE cb_type_field_relation AS A INNER JOIN cb_type_field_relation AS B ON A.cb_type_id = B.cb_type_id
+    $query = "UPDATE cb_type_field_relation AS A INNER JOIN cb_type_field_relation AS B ON A.cb_type_id = B.cb_type_id
         SET A.`order_display` = 8 
-        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname = 'buffering_timeout')"
-    );
+        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname = 'buffering_timeout')";
+    if ($pearDB->query($query) === false) {
+        throw new \PDOException(
```

Comment:
The CentreonDB class is already configured to generate an exception if an error occurs.